gateway/api/views/programs.py

```diff
@@ -104,45 +101,6 @@ def get_serializer_class(self):
     def get_object(self):
         logger.warning("ProgramViewSet.get_object not implemented")
 
-    def get_run_queryset(self):
```

Comment:
I needed to apply this change in this PR due to a bug in this method. The change here was basically:
- remove `get_run_queryset`
- use the function repository class [here](https://github.com/Qiskit/qiskit-serverless/pull/1571/files#diff-60276657248dc9f5babc8ac88d48a0709ec594e741be0bc0b12103419e37a5d6R212)